frame/balances/src/lib.rs

```diff
@@ -1150,6 +1157,7 @@ impl<T: Trait<I>, I: Instance> ReservableCurrency<T::AccountId> for Module<T, I>
 		Self::try_mutate_account(who, |account, _| -> DispatchResult {
 			account.free = account.free.checked_sub(&value).ok_or(Error::<T, I>::InsufficientBalance)?;
 			account.reserved = account.reserved.checked_add(&value).ok_or(Error::<T, I>::Overflow)?;
+			Self::deposit_event(RawEvent::Reserved(who.clone(), value.clone()));
```

Comment:
These deposit events mean we need to change the weight formulas?

Reply:
Possibly, @shawntabrizi ?

Reply:
No, events are free for any particular dispatch.

We already know that this storage item will be read from and written to once per block, and the new events append api keeps the cost minimal.